docs/markdown/ios/toast.md

```diff
@@ -0,0 +1,158 @@
+
+## Best practices
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/b5/86/15/b5861579a1c5f7b958c6b7a7099ee9f4.jpg" alt="example with toast with unblocking placement"/>
+    <Do title="Do" />
+    Place Toasts out of the way so that a user can still navigate and complete tasks. Keep a bottom margin that is the same size as the left and right margins.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/f1/9c/d1/f19cd161c8cc6112a450fb58e0dc03d2.jpg" alt="example of with blocking placement"/>
+    <Dont title="Don't" />
+    Block navigation controls with Toasts or align too close to the edge of a navigation bar.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/5b/49/ce/5b49ce5d555fdd56b3b555fb62fcaaea.jpg" alt="example of showing one toast"/>
+    <Do title="Do" />
+    Show one Toast at a time, with errors and Acknowledgements taking priority.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/e2/c7/4e/e2c74eed170be2f7029a673ef4b8512e.jpg" alt="example of showing multiple toasts"/>
+    <Dont title="Don't" />
+    Stack multiple toasts as that will block the user.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/36/0a/c3/360ac34d4f9b31c22058a056abd89e21.jpg" alt="example of a dismissible toast"/>
+    <Do title="Do" />
+    Include a way to dismiss the toast when it is actionable or contains multiple lines of text. Mobile toasts can be dismissed via swiping on the toast in any direction.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/c8/ea/51/c8ea51c06c88110bde0f2854e9103980.jpg" alt="example of leaving a toast on screen"/>
+    <Dont title="Don't" />
+    Leave toasts on screen for a long time without a way to dismiss. Exceptions are blocking error toasts where a user can’t take any action until the error is resolved.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+People use Apple’s accessibility features, such as reduced transparency, VoiceOver, and increased text size to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple’s Human Interface Guidelines and documentation about accessibility for iOS:
+
+[Accessbile design on Android](https://material.io/design/usability/accessibility.html#understanding-accessibility)
+[Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility)
+
+### Duration
+Some people may take longer to read toasts than others due to aging or other cognitive impairments Use the guide below to set duration for Toasts:
+
+- Brief text of approximately 10–15 words (including button text): 5s
+- Longer than 15 words: Slow readers can read about 125–200 words per minute. Base your duration on the slowest number. For example, a toast with 20 words should be set to 10s. [Source](https://capitalizemytitle.com/reading-time/3000-words/)
```

Comment:
I would spell out seconds instead of just the s

Reply:
Done!